### PR TITLE
feat: add integration for scrapegraphai

### PR DESCRIPTION
**Pull request by VinciGit00**
I added the integration as a tool for scrapegraph ([link](https://scrapegraphai.com))

### CODE REVIEW

File: llama-index-integrations/tools/llama-index-tools-scrapegraph/readme.md
Comment (logan-markewich) [inline llama-index-integrations/tools/llama-index-tools-scrapegraph/readme.md:1]: should be renamed `README.md` fyi

**Comment by logan-markewich** [comment]
I don't have permissions to push. You are missing a build file though, in the examples dir

**Comment by VinciGit00** [comment]
Hi @logan-markewich , I created the build files and renamed the README.md from readme.md

**Comment by VinciGit00** [comment]
Hi @logan-markewich ltmk if everything is ok

File: llama-index-integrations/tools/llama-index-tools-scrapegraph/readme.md
Comment (logan-markewich) [reply llama-index-integrations/tools/llama-index-tools-scrapegraph/readme.md:1]: Can we rename the readme to be uppercase? `README.md` ?

**Comment by logan-markewich** [comment]
Maybe @VinciGit00 can you give me write permissions to this branch? Or open it from a more public branch/account? I think it will be faster if I can push the changes needed to fix CICD

**Comment by VinciGit00** [comment]
Hi @logan-markewich I added you the permissions for write access, lmk if you need something

**Comment by VinciGit00** [comment]
Hi @logan-markewich I added you the permissions for write access, lmk if you need something

**Comment by logan-markewich** [comment]
Closing in favor of https://github.com/run-llama/llama_index/pull/17238

**Pull request closed without merging**